tower_eval/cli.py

```diff
@@ -154,6 +157,11 @@ def run_evaluations(configs: dict) -> dict:
                     eval_args = combine_metrics_args(
                         task_metric_args, subtask_metric_args
                     )
+                    if subtask_metric_args:
```

Comment:
what happens if we have only a single subtask with specific args in the middle of all the subtasks? I think from that subtask on we will keep the metric with those specific args, right?

For example, in the config below, what will be the arguments for evaluating `booking_tickets-dynamic.en-fr` and `classpass_chat-dynamic.en-de`?

```
tasks:
  - name: mt
    subtasks:
      aboutyou-ticket-dynamic.en-de:
      am_eagle_chat-dynamic.en-fr:
      apple_apple_chat_for_core_chat-dynamic.en-it:
      belkin_chat-dynamic.en-ko:
        eval_args:
          metrics:
            bleu:
              tokenizer: ko-mecab
      belkin_chat-dynamic.en-zh-CN:
        eval_args:
          metrics:
            bleu:
              tokenizer: zh
      booking_tickets-dynamic.en-fr:
      classpass_chat-dynamic.en-de:
```